examples/PyTest/test_plan.py

```diff
@@ -1,14 +1,30 @@
 #!/usr/bin/env python
 # This plan contains tests that demonstrate failures as well.
 """Example to demonstrate PyTest integration with Testplan."""
-import sys
 import os
+import sys
 
 from testplan import test_plan
+from testplan.common.utils.context import context
 from testplan.testing import py_test
-
 from testplan.testing.multitest.driver.tcp import TCPServer, TCPClient
-from testplan.common.utils.context import context
+
+
+def before_start(env, result):
+    result.log("Executing before start hook.")
+
+
+def after_start(env, result):
+    result.log("Executing after start hook.")
+    env.server.accept_connection()
```

Comment:
This example had an environment but did absolutely nothing with it so added at least one call to the env.